specs/phase1/shard-transition.md

```diff
@@ -44,17 +44,33 @@ def compute_shard_transition_digest(beacon_state: BeaconState,
 
 ```python
 def verify_shard_block_message(beacon_state: BeaconState,
-                               shard_state: ShardState,
+                               shard_parent_state: ShardState,
```

Comment:
Similar, we could consider _not_ passing in `shard_parent_state` explicitly because it is a function of `(beacon_state, shard)`. This could prevent accidental misuse of the function

Reply:
also due to  fail-crosslinking, we can't use `beacon_state.shard_states[shard_store.shard]`. :'(